config/locales/en.yml

```diff
@@ -116,6 +117,20 @@ en:
       # html: '<abbr title="required">*</abbr>'
     error_notification:
       default_message: "Please review the problems below:"
+    hints:
+      trace:
+        tagstring: comma delimited
+    labels:
+      trace:
+        gpx_file: Upload GPX File
+        visibility: Visibility
+    options:
+      trace:
+        visibility:
+          private: Private (only shared as anonymous, unordered points)
+          public: Public (shown in trace list and as anonymous, unordered points)
+          trackable: Trackable (only shared as anonymous, ordered points with timestamps)
+          identifiable: Identifiable (shown in trace list and as identifiable, ordered points with timestamps)
```

Comment:
With these descriptions moved here do we still need the old ones?

Reply:
Possibly - I think the existing translations are used for the `:show` action. That could be refactored to use the hints.